source/fem-dem/gls_sharp_navier_stokes.cc

```diff
@@ -249,8 +254,8 @@ GLSSharpNavierStokesSolver<dim>::generate_cut_cells_map()
               size_t id;
               for (unsigned int j = 0; j < local_dof_indices.size(); ++j)
                 {
-                  id               = local_dof_indices[j];
-                  vertices_cut[id] = {true, particle_id_which_cuts_this_cell};
+                  id                            = local_dof_indices[j];
+                  local_dof_over_constraint(id) = 1;
```

Comment:
can you give a better variable name over_constraint you mean over_constrained no?

Reply:
You're right. Is dof_overconstrained fine?